src/components/FlowRunBreadCrumbs.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="flow-run-bread-crumbs">
+    <FlowRouterLink :flow-id="flowRun.flowId" class="flow-run-bread-crumbs__flow-link">
+      <template #after>
+        <p-icon icon="ChevronRightIcon" size="small" />
+      </template>
+    </FlowRouterLink>
+    <p-link :to="routes.flowRun(flowRun.id)" class="flow-run-bread-crumbs__flow-run-link">
+      <span>{{ flowRun.name }}</span>
+    </p-link>
+  </div>
+</template>
+
+<script lang="ts" setup>
+  import FlowRouterLink from '@/components/FlowRouterLink.vue'
+  import { useWorkspaceRoutes } from '@/compositions/useWorkspaceRoutes'
+  import { FlowRun } from '@/models'
+
+  defineProps<{
+    flowRun: FlowRun,
+  }>()
+
+  const routes = useWorkspaceRoutes()
+</script>
+
+<style>
+.flow-run-bread-crumbs { @apply
+  text-slate-700
```

Comment:
```suggestion
  text-foreground-600
```
I think you're just looking for bolded text here and not this color specifically, right?

Reply:
This is the color that the `StateListItem` uses which is where this was getting its color from. Adding it here so this bread crumb looks as it does currently in the list item in the popover. Not intending to change any styles. This applies to the task run list item as well. 

![image](https://user-images.githubusercontent.com/6200442/226748603-e08090d5-d4c3-42e6-b549-96397adf2db7.png)

Reply:
foreground-600 is to bright IMO. 
![image](https://user-images.githubusercontent.com/6200442/226748887-95bd3ee9-4234-4996-8cc5-0ffad19ebb49.png)

200 looks good to me. I could change both this component and the StateRunListItem if you'd like
![image](https://user-images.githubusercontent.com/6200442/226749071-73493d78-b9f2-416e-a202-a852ec0b4ddd.png)

Reply:
Oh it's wrong there too lol - we just haven't noticed because all the important text there is links that are highlighted in blue

Reply:
Yes please, that looks great

Reply:
Updated 👍